content/en/install.md

```diff
@@ -3,101 +3,206 @@ title: Installation
-across the entire computer, often have older versions, and don\'t have
-as many available versions. Source compilation is much more difficult
-but is necessary for debugging and development. If you don\'t know which
-installation method you need or prefer, we recommend the Scientific
-Python Distribution [Anaconda](https://www.anaconda.com/download/).
+To install SciPy with [static type stubs],
+see [Installing with type stubs](#type-stubs).
 
-<a name="distributions"></a>
+[static type stubs]: https://typing.readthedocs.io/en/latest/guides/libraries.html
 
-# Scientific Python Distributions (recommended)
+<a name="project-based"></a>
 
-Python distributions provide the language itself, along with the most
-commonly used packages and tools. These downloadable files require
-little configuration, work on almost all setups, and provide all the
-commonly used scientific Python tools.
+## Project-based workflows
 
-[Anaconda](https://www.anaconda.com/download/) works on Windows, Mac,
-and Linux, provides over 1,500 Python packages, and is used by over 15
-million people. Anaconda is best suited to beginning users; it provides
-a large collection of libraries all in one.
+In project-based workflows, a project is a directory containing a manifest
+file describing the project, a lock-file describing the exact dependencies
+of the project, and the project's (potentially multiple) environments.
 
-For more advanced users who will need to install or upgrade regularly,
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge) is a more
-suitable way to install the `conda` (and `mamba`, a faster `conda` alternative)
-package manager.
+### Installing with `uv`
 
-Other options include:
+{{< admonition tip >}}
+If it is your first time installing a Python library, start here!
+{{< /admonition >}}
 
-- [WinPython](https://winpython.github.io): Another free distribution
-  including scientific packages and the Spyder IDE; Windows only, but
-  more actively maintained and supports the latest Python 3 versions.
-- [Pyzo](https://pyzo.org): A free distribution based on Anaconda
-  and the IEP interactive development environment; Supports Linux,
-  Windows, and Mac.
+Here is a step-by-step guide to setting up a project to try SciPy with [`uv`],
+a Python package manager.
 
-<a name="pip-install"></a>
+[`uv`]: https://docs.astral.sh/uv/
 
-# Installing with Pip
+1. Install `uv`, following [the instructions in the `uv` documentation][install-uv].
 
-You can install SciPy from PyPI with `pip`:
+[install-uv]: https://docs.astral.sh/uv/getting-started/installation/
 
-    python -m pip install scipy
+2.  Create a new project in a new subdirectory, by executing the following in a terminal:
 
-<a name="conda-install"></a>
+        uv init try-scipy
+        cd try-scipy
 
-# Installing via Conda
+    {{< admonition hint >}}
+The second command changes directory into the directory of your project.
+    {{< /admonition >}}
```

Comment:
@jarrodmillman, I believe the linter is suggesting changes which would break the formatting.

Reply:
You could try something like
```
<!-- prettier-ignore-start -->
...
<!-- prettier-ignore-end -->
```